apps/services/user-profile/src/app/v2/test/me-user-profile.controller.spec.ts

```diff
@@ -179,6 +182,7 @@ describe('MeUserProfileController', () => {
         user: createCurrentUser({
           nationalId: testUserProfile.nationalId,
           scope: [UserProfileScope.read, UserProfileScope.write],
+          audkenniSimNumber: audKenniPhoneNumber,
```

Comment:
~ If you rename the variable above.
```suggestion
          audkenniSimNumber,
```